lib/tools/android-manifest.js

```diff
@@ -89,18 +89,6 @@ manifestMethods.packageAndLaunchActivityFromManifest = async function (localApk)
   }
 };
 
-manifestMethods.targetSdkVersionFromManifest = async function (localApk) {
```

Comment:
another option would be to attempt to retrieve the target sdk version, but simply ignore the error in the calling function. something like this:

```js
let targetSdk = null;
try {
  targetSdk = await this.targetSdkVersionFromManifest(apk);
} catch (e) {
  logger.warn(`Ran into problem getting target SDK; ignoring: ${e.message}`);
}
if (apiLevel >= 23 && (!targetSdk || targetSdk >= 23)) {
```